modules/muuntaja/src/muuntaja/core.clj

```diff
@@ -76,22 +78,39 @@
 ;; default options
 ;;
 
+(defn header
+  "Extract a specific header from a request or response map.
+  This has to be done in a case-insensitive manner for http-1.1.
+  With http-2 header names need to be lower-cased before encoding."
+  [^String header-name request-or-response]
+  (some (fn [[n v]]
+          (when (.equalsIgnoreCase header-name n)
+            v))
+        (:headers request-or-response)))
+
+(defn header?
```

Comment:
Could be defined as `(some? (header header-name request-or-response))`. Current implementation will return the supplied `header-name` or `nil`.